src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/evaluators/CompOverlap.java

```diff
@@ -8,15 +8,6 @@
 import org.broadinstitute.hellbender.tools.walkers.varianteval.util.Analysis;
 import org.broadinstitute.hellbender.tools.walkers.varianteval.util.DataPoint;
 
-/**
- * The Broad Institute
```

Comment:
I just did a check and these 3 files seem to be the last vestiges of old license code in the headers of any of our source files.